docs/fundamentals/actors/create-actor/test.mdx

```diff
@@ -10,7 +10,7 @@ import TabItem from "@theme/TabItem";
 
 Now that your actor project is generated we can write a simple unit test to verify its functionality. We want to ensure that our logic to parse the query string for a name and return `Hello <name>` is correct.
 
-Right now, our actor is laid out in a way that is difficult to test properly. Let's look at the function signature inside of the `impl` block:
+Let's revert our actor to the simpler initial version. Our actor is laid out in a way that is difficult to test properly. Let's look at the function signature inside of the `impl` block:
```

Comment:
I don't think we're reverting here but I do like the idea of rewording this.
```suggestion
Our actor code is currently organized in a way that is difficult to test properly. Let's look at the function signature inside of the `impl` block:
```

Reply:
> I don't think we're reverting here but I do like the idea of rewording this.

The reason I chose that specific working is that, in the previous section, the actor code is modified to include another query param - "msg". But the code in this section is that of the original single param functionality. Could move the msg code into this section and modify the test?

Reply:
Ah, I see. This feels like a good way to phrase this section then and we can revisit later (really, we just need a bit of a better testing strategy!)